python/core/meet-python/python-2.md

```diff
@@ -51,6 +52,18 @@ Python 3 is considered to be an improved version of Python 2, with fewer mistake
 
 As a beginner, your best bet is **Python 3**. This is the standard that's advancing slightly faster and you don't have to worry at all about porting a whole Python 2 application to Python 3 (the main reason why Python 2 hung around for so long).
 
+---
+## Practice
+
+To print to console in python 3 we use a:
+
```

Comment:
rewrite this question, doesn't really discuss what the article covers -
what objective is this intended to cover?